scripts/exrrfs_run_prepstart.sh

```diff
@@ -389,7 +385,7 @@ else
   if [ -r "${checkfile}" ] ; then
     cp ${bkpath}/${restart_prefix}coupler.res      bk_coupler.res
     cp ${bkpath}/${restart_prefix}fv_core.res.nc   fv_core.res.nc
-    if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    if [ "${IO_LAYOUT_Y}" = "1" ]; then
       for file in ${filelistn}; do
         if [ ${cycle_subtype} == "spinup" ] ; then
```

Comment:
Should be "CYCLE_SUBTYPE" for line 390 and 402

Reply:
Thanks for finding these! Fixed.